recipes/shortstack/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set name = "ShortStack" %}
+{% set version = "3.8.3" %}
+{% set sha256 = "7c76f51ed949ca95ec5df7cb54803ae2350658fd64c75909351d5a856abb0dbe" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/MikeAxtell/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+
+requirements:
+  run:
+    - perl
+    - samtools 1.*
+    - bowtie 1.*
+    - viennarna 2.*
+
+test:
+  commands:
+    - ShortStack --help
+    - ShortStack --version
+
+about:
+  home: https://github.com/MikeAxtell/ShortStack
+  license: GPL-3.0
+  license_family: GPL
+  license_file: LICENSE
+  summary: 'ShortStack: Comprehensive annotation and quantification of small RNA genes'
+
+extra:
+  recipe-maintainers:
+    - sebastian-luna-valero
```

Comment:
Bioconda doesn't really have this setting. It's used in conda-forge to initialize the permissions for the feedstock repositories.